types/interface.go

```diff
@@ -3,7 +3,19 @@ package types
 import (
 	"time"
 
-	"github.com/multiversx/mx-chain-core-go/storage"
+	"github.com/multiversx/mx-chain-core-go/data"
+)
+
+// RemovalStatus removal status defines the removal possibility of a key
+type RemovalStatus string
+
+const (
+	// UnknownRemovalStatus defines the unknown status
+	UnknownRemovalStatus RemovalStatus = "unknown"
+	// ExplicitlyRemovedStatus defines the explicitly removed status
+	ExplicitlyRemovedStatus RemovalStatus = "explicitly removed"
+	// NotRemovedStatus the key is not removed
```

Comment:
// NotRemovedStatus defines the not removed status ?

Reply:
fixed